samples/shields/x_nucleo_iks01a1/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _x-nucleo-iks01a1-sample:
+
+X-NUCLEO-IKS01A1: MEMS Interial and Environmental Multi sensor shield
+#####################################################################
+
+Overview
+********
+This sample enables all sensors of X-NUCLEO-IKS01A1 shield by reading
+and displaying periodically data from shield sensors:
+
+ - HTS221: Temperature and humidity
+
+ - LPS25HB: Atmospheric pressure
+
+ - LIS3MDL: 3-axis Magnetic field intensity
+
+ - LSM6DSL: 3-Axis Acceleration
+
+Requirements
+************
+
+This sample uses the X-NUCLEO-IKS01A1 shield stacked on a board over
```

Comment:
This sample communicates over I2C with the X-NUCLEO-IKS01A1 shield
    stacked on a board with an Arduino connector. The board's I2C must be
    configured for the I2C Arduino connector (both for pin muxing
    and device tree).